Add readme example project (#450)
tokio-rs/axum: README.md

    // run our app with hyper
    // `axum::Server` is a re-export of `hyper::Server`
    let addr = SocketAddr::from(([127, 0, 0, 1], 3000));

>     tracing::info!("listening on {}", addr);

Reviewer: I don't see much reason to change this and its inconsistent with the other example. I think we should just keep it like it was.

Author: Sure, I thought it was just a little more new user friendly since it doesn't print anything by default with the suggested cargo run

Either way is fine with me

}
```

You can find this [example][readme-example] as well as other example projects in the [example directory][examples].

> There are also a number of usage examples in the [crate documentation][docs].

Reviewer: I think the previous wording is fine.

```suggestion
See the [crate documentation][docs] for way more examples.
```
